### ISSUE TITLE
Retrieving Empty Ammo Crates from Logistics Crate

### ISSUE THREAD

**Issue by AlphaKilo07** [Vdauphin/HeartsAndMinds#325]
It'd be immensely useful if there were a way to spawn in a crate empty. Perhaps add empty crates here:
https://github.com/Vdauphin/HeartsAndMinds/blob/ae90393b0607a1439479d7896e7d89503be5a6dc/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/core/def/mission.sqf#L310

**Comment by Vdauphin** [comment]
Hello,
Ammo crate are not always full. You can still add stuff you want. Nop?
Best regards.

**Comment by AlphaKilo07** [comment]
I'll specifiy more to why I posted this. I posted this at around 3AM haha. 

I wanted to spawn a NATO ammo box which is small so I could transport it easily to the battlefield in a little bird, but the small NATO ammo box was full afaik.

I could make the commit but I don't know if there's any empty ammo box. Perhaps there can be a ace interaction to empty ammo (would call delete function since there there might be too many things on the ground) or modify the GUI with an empty box when selecting storage/ammunition (GUIs are a bitch though).
